src/app/http-stalker.service.ts

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import {Injectable} from '@angular/core';
+import {User} from './user';
+import {HttpClient, HttpHeaders} from '@angular/common/http';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class HttpStalkerService {
+  httpOptions: any = {
+    headers: new HttpHeaders({'Content-Type': 'application/json'}),
+    observe: 'response',
+  };
+  constructor(public http: HttpClient) {}
+  fakepost(apiURL: string, user: User): any {
+    return this.http.post(apiURL, user, this.httpOptions);
+  }
```

Comment:
come mai questo nome? nel codice la post non dovrebbe mai venire fatta "per finta", è compito della dep in-memory-web-api di intercettare le richieste nei test. in teoria lo fa pure automaticamente!

Reply:
Si è un refuso dei nostri tentativi precedenti ma avevo intenzione di sistemare queste cose stasera